src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -24,6 +24,7 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonView;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
```

Comment:
This import is not used in this file.

Reply:
Removed in 4ac2e4cab7a40d0c40960ba15503c2781adef26d